libcxx/test/support/nasty_macros.h

```diff
@@ -59,8 +59,8 @@
 #define __out NASTY_MACRO
 #endif
 
-#define __output NASTY_MACRO
```

Comment:
We should keep these to reduce the diff to upstream and ensure that libc++ does not introduce __input/__output names.

The __cheri_* are not required here since it's unlikely anyone would use that as a variable/function name.

Reply:
I'd be tempted to not add them since that reduces the diff to upstream.

Reply:
Sounds good.